### ISSUE TITLE
prompts / data

### ISSUE THREAD

**Issue by jonschlinkert** [generate/generate#38]
I recently introduced a bug that caused the license statement intended for a readme to be used as the `license` in package.json. 

The bad news: I just pushed up a temporary fix for that bug, but it's not unlikely that other bugs will crop up as a result.

The good news: Data handling in generate is highly modular and the code for it is organized in a way that makes it easy to completely change. I'm working on that now. If anyone experiences regressions, please list them here. Sincerest apologies for any inconvenience

**Comment by dawsonbotsford** [comment]
Link to fix? It'd be cool to learn and help prevent this type of bug in the future.

**Comment by jamen** [comment]
I've experienced this bug recently too, just for the record.  Was able to fix for myself just by regenerating.

**Comment by jonschlinkert** [comment]
> I've experienced this bug recently too, just for the record. (same one involving license)

Good to know. I attempted a fix yesterday, you'll probably need to reinstall global generators and generate

> Link to fix? It'd be cool to learn and help prevent this type of bug in the future.

I started working on a fix for this one a week or so ago. Let me try to get that wrapped up, then we can do a post-mortem. We should probably also open an issue here to discuss strategies for improving how data is handled by generate.  sound ok?
